hungry.py

```diff
@@ -1,5 +1,5 @@
 inp = input("Are you hungry ? ")
-if(in.lower() == "yes"):
+if(in.lower() == "yes" or in.lower() == "y"):
```

Comment:
great, please add else block

Reply:
ok

Reply:
and also please write one line for loop example